lib/cdo/i18n_string_url_tracker.rb

```diff
@@ -119,14 +127,14 @@ def flush
     # If the DCDO flag has changed since data was buffered, we want to clear the buffer and not log/flush the data.
     return unless DCDO.get(I18N_STRING_TRACKING_DCDO_KEY, false)
 
-    # log every <string_key>:<url>:<source> combination to Firehose
+    # log every <url>:<normalized_key>:[source, string_key, scope, separator] combination to Firehose
```

Comment:
Each of these are a separate column in the database, right? 
```suggestion
    # log every <url>:<normalized_key>:<source>:<string_key>:<scope>:<separator> combination to Firehose
```